Realm/RLMResults.h

```diff
@@ -150,6 +150,67 @@ RLM_ASSUME_NONNULL_BEGIN
  */
 - (RLMResults RLM_GENERIC_RETURN*)sortedResultsUsingDescriptors:(NSArray *)properties;
 
+#pragma mark - Asynchronous Queries
+
+/**
+ Asynchronously run this query and then pass the result to the block on the given dispatch queue.
+
+ The block will be called again after each write transaction which changes the
+ result of the query. You must retain the returned token for as long as you want
+ the results to continue to be sent to the block. To stop receiving updates,
+ call -stop on the token.
+
+ The determination for if the results have changed is currently very coarse, but
+ will become more precise in future versions.
+
+ If an error occurs the block will be called with `nil` for the results
+ parameter and a non-`nil` error. Currently the only errors that can occur are
+ when opening the RLMRealm on the background worker thread or the destination
+ queue fails.
+
+ The RLMResults passed to the block is a normal auto-updating RLMResults object.
+ This means that if you perform a write transaction or explicitly call
+ -[RLMRealm refresh] from within the block, the query may be synchronously rerun
+ the next time you access the results.
+
+ @warning This method cannot be called during a write transaction, or when the
+          containing realm is read-only.
```

Comment:
Handover is only implemented for SharedGroup since it uses the transaction version.